src/main/java/org/apache/hadoop/fs/PrestoFileSystemCache.java

```diff
@@ -217,6 +221,11 @@ private static boolean isHdfs(URI uri)
         return "hdfs".equals(scheme) || "viewfs".equals(scheme);
     }
 
+    private static boolean isExtraCredentialChanged(FileSystem fileSystem, Configuration configuration)
```

Comment:
This method sounds strange from a grammar perspective. How about `extraCredentialsChanged`?